deployment-configuration/compose/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
I see your point, but trying to share the files between the helm configuration files and the docker compose one will only make things more obscure to understand. 

Those two files (this one and the `values.yaml`) that are in the `deployment-configuration` needs to be duplicated in a way. If not, it means that there is a weird behavior that we should implement saying that the content of those files that are required for the template generation to work, need to be took from the `helm` folder for some reasons. It also means that we cannot rely on the current mechanism which copies files for a target from the corresponding folder, but instead, we would have a mix between "I pick files from the dedicated `compose` folder, but sometimes no, I need to pick them in the `helm` folder", which makes the generator more complicated and the logic of the configuration file generation more obscure. 

A lot of files appears to be duplicated, but they are here to support the simplicity of "when I generate a target, the files from the corresponding target folder are copied" instead of having a mix of "sometimes its took from this folder, sometimes no", which would also require to implement a mechanism to check if the files that needs to be picked from the `helm` folder needs to be copied only if it doesn't exists in the `compose` folder to support configuration overriding (or other target folder if we support others in the future). 

Another solution would be to share those files in a `common` or `shared-configuration` folder which would require to implement the same logic described above, but with another folder. I would vote against it as it makes the whole logic more complicated to understand, makes the generator more complex than just "copy files from the corresponding folder" and breaks, in a way, the isolation and independence of each target, to avoid having two files that looks the same. Why make it complicated when we can keep it simple and stupid?

Reply:
I see your point the reason to avoid duplication. I'm removing the irrelevant configuration from compose's values.yaml